src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule232specialescape/InputIllegalTokenText.java

```diff
@@ -14,32 +14,32 @@ public void methodWithLiterals()
 
     public String wrongEscapeSequences()
     {
-        final String r1 = "\u0008"; //ok
+        final String r1 = "\u0008";  
         final String r2 = "\u0009"; //warn
         final String r3 = "\u000csssdfsd"; //warn
         final String r4  = "\u1111sdfsd\444"; // ok
 
         final char r5 = '\012'; //warn
         final char r6 = '\u0022'; //warn
-        final char r7 = '\b'; //ok
+        final char r7 = '\b';  
         return "\u000csssdfsd"; //warn
     }
 
     public void specialCharsWithoutWarn()
     {
-        String r1 = "\b"; //ok
-        String r2 = "\t"; //ok
-        String r3 = "\n"; //ok
-        String r4 = "\f"; //ok
-        String r5 = "\r"; //ok
-        String r6 = "\""; //ok
-        String r7 = "\'"; //ok
-        String r8 = "\\"; //ok
+        String r1 = "\b";  
```

Comment:
```suggestion
        String r1 = "\b"; 
```
There shouldn't be any trailing white-space at the end of line after removing the comment. Apply the changes for all such occurrence.